app/views/bulkrax/shared/_entries_tab.html.erb

```diff
@@ -1,44 +1,16 @@
-<div role="tabpanel" class="tab-pane bulkrax-nav-tab-table-left-align <%= pagination_anchor == 'work-entries' ? 'active' : '' %>" id="<%= pagination_anchor %>">
-  <table class='table table-striped'>
+<div role="tabpanel" class="tab-pane bulkrax-nav-tab-table-left-align" >
```

Comment:
There's no declared `tablist` nor `tab` roles, so I think we should not have a `tabpanel`

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role